packages/components/src/components/button/button.tsx

```diff
@@ -201,6 +201,12 @@ export class Button {
       this.disabled && 'disabled'
     );
 
+    if (!this.disabled) {
```

Comment:
IMHO we can achieve the same by using the Stencil [reflect ](https://stenciljs.com/docs/properties#reflect-properties-values-to-attributes-reflect)attribute for the 'disabled' prop